api/FormData.json

```diff
@@ -0,0 +1,1196 @@
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "12"
+          },
+          "opera_android": {
+            "version_added": "12"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "append": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FormData/append",
+          "support": {
+            "webview_android": {
+              "version_added": "3",
+              "notes": "XHR in Android 4.0 sends empty content for FormData with blob."
+            },
+            "chrome": {
+              "version_added": "7"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "4",
+              "notes": "Prior to Gecko 7.0 (Firefox 7.0 / Thunderbird 7.0 / SeaMonkey 2.4), if you specified a <a href='https://developer.mozilla.org/docs/Web/API/Blob'><code>Blob</code></a> as the data to append to the object, the filename reported in the 'Content-Disposition' HTTP header was an empty string; this resulted in errors being reported by some servers. Starting in Gecko 7.0 the filename 'blob' is sent."
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "notes": "Prior to Gecko 7.0 (Firefox 7.0 / Thunderbird 7.0 / SeaMonkey 2.4), if you specified a <a href='https://developer.mozilla.org/docs/Web/API/Blob'><code>Blob</code></a> as the data to append to the object, the filename reported in the 'Content-Disposition' HTTP header was an empty string; this resulted in errors being reported by some servers. Starting in Gecko 7.0 the filename 'blob' is sent."
+            },
+            "ie": {
+              "version_added": "10",
+              "notes": "With the'Include local directory pass when uploading files to a server' option enabled, IE will change the filename inside the <a href='https://developer.mozilla.org/docs/Web/API/Blob'><code>Blob</code></a> on the fly. To have direct control of the sent filename, the developer should send the filename as the third parameter value, i.e. <i>formData.append(name, value, filename)</i>."
```

Comment:
A couple little quirks in the formatting here. It looks like there's a space missing before "Include" and the example call at the end might read a little nicer in `<code>` instead of `<i>`. Also, instead of single quotes around the option text, you can use double quotes with a backslash, like this `\"`, or an HTML entity (like `&quot;`)